### PR TITLE
Scripts output to stdout

### PR DESCRIPTION
**Pull request by richsalz**
And only generate one output "file" at a time for objects.pl (That is, we run it twice, with different flags, to generate the different output files.)

### CODE REVIEW

File: Configurations/unix-Makefile.tmpl [hunk 1]
```diff
@@ -646,13 +646,18 @@ generate_crypto_bn:
 	( cd $(SRCDIR); $(PERL) crypto/bn/bn_prime.pl > crypto/bn/bn_prime.h )
 
 generate_crypto_objects:
+	( cd $(SRCDIR); $(PERL) crypto/objects/objects.pl -n \
+				crypto/objects/objects.txt \
+				crypto/objects/obj_mac.num \
+				>crypto/objects/obj_mac.new && \
```
Comment (levitte) [inline Configurations/unix-Makefile.tmpl]: Space after `>`?

File: Configurations/unix-Makefile.tmpl [hunk 2]
```diff
@@ -646,13 +646,18 @@ generate_crypto_bn:
...
+	    mv crypto/objects/obj_mac.new crypto/objects/obj_mac.num )
 	( cd $(SRCDIR); $(PERL) crypto/objects/objects.pl \
 				crypto/objects/objects.txt \
 				crypto/objects/obj_mac.num \
-				include/openssl/obj_mac.h )
+				>include/openssl/obj_mac.h )
```
Comment (levitte) [inline Configurations/unix-Makefile.tmpl]: Here as well...

Comment (richsalz) [reply Configurations/unix-Makefile.tmpl]: additional commit, to be squashed, to add the space for consistency.

**Comment by richsalz** [comment]
ping @levitte

**Comment by richsalz** [comment]
merged, thanks!

**Pull request closed without merging**